### ISSUE TITLE
The training results are good, but the test results are very poor.

### ISSUE THREAD

**Issue by abner2015** [ThomasDelteil/HandwrittenTextRecognition_MXNet#26]
when i use my dataset train the ocr model , the training results are good, but the test results are very poor. i do not need segment the roi . the train data is number .how can i improve the test result. 
![3](https://user-images.githubusercontent.com/13195667/54362007-26700580-46a3-11e9-8648-9e1ef6445f54.png)

**Comment by ThomasDelteil** [comment]
It looks like you are experiencing overfitting. https://en.wikipedia.org/wiki/Overfitting
A few ideas to overcome this and regularization to your model:
- you can stop training when the validation loss start going up for N epochs
- you can add more data augmentation on your training set
- you can increase the size of your training set
- you can add weight decay in your optimizer
- you can add more dropout layer / tune the dropout values
- you can use a different learning rate schedule
- you can use more batch normalization layers

**Comment by samar-smida** [comment]
@ThomasDelteil 
I am testing your project but I have an assertion error even though I put my email and password in the credentials.json .
In the registration form, they ask for the email and not the username.
I found this link in the project to have but it is not functional https://fki.tic.heia-fr.ch/DBs/iamDB/iLogin/index.php 
please can you help me
